romanisim/parameters.py

```diff
@@ -84,6 +84,17 @@
     "linearity": None,
     "readnoise": 5.0 * u.DN,
     "saturation": 55000 * u.DN,
+    # Taken from roman_wfi_photom_0046.asdf
+    "photom": {"pixelareasr": {"F062": 2.640600009241359e-13 * u.sr,
+                               "F087": 2.640600009241359e-13 * u.sr,
+                               "F106": 2.640600009241359e-13 * u.sr,
+                               "F129": 2.640600009241359e-13 * u.sr,
+                               "F146": 2.640600009241359e-13 * u.sr,
+                               "F158": 2.640600009241359e-13 * u.sr,
+                               "F184": 2.640600009241359e-13 * u.sr,
+                               "F213": 2.640600009241359e-13 * u.sr,
```

Comment:
This looks like it's only used in the tests; let's move it there if we need this?  I think my preference would be that users mostly use the WCS to figure out the pixel scale.

Reply:
This is used in other L3 code, so should stay.